docs/weld.md

```diff
@@ -0,0 +1,56 @@
+# Bootstraping a snappy device
+
+The `snap weld` command is designed to do the snap specific part
+of building an image.
+
+## Running it
+
+The `snap weld` command takes a model assertion as intput and
+also `--root-dir` and `--gadget-unpack-dir`. It will place
+all snaps in the root-dir and downloads/unpacks the gadget
+snap into the directory specified with `--gadget-unpack-dir`.
```

Comment:
For the command-line use: `snap prepare-image foo.yaml /target/dir` and in target-dir there will be `./target-dir/{gadget,image}`.